default/scripting/species/common/industry.macros

```diff
@@ -5,15 +5,25 @@ BASIC_INDUSTRY
 '''        EffectsGroup
             scope = Source
             activation = And [
-                Source
                 Planet
                 TargetIndustry low = 0
                 Focus type = "FOCUS_INDUSTRY"
             ]
             accountinglabel = "FOCUS_INDUSTRY_LABEL"
-            priority = [[EARLY_PRIORITY]]
+            priority = [[DEFAULT_PRIORITY]]
```

Comment:
@MatGB, did you get around to fix that issue as you intended? I don't recall seeing any commits regarding this...